tests/ui/PureReportActionItemTest.tsx

```diff
@@ -107,7 +107,7 @@ describe('PureReportActionItem', () => {
             await waitForBatchedUpdatesWithAct();
 
             expect(screen.getByText(actorEmail)).toBeOnTheScreen();
-            expect(screen.getByText(translateLocal('iou.automaticallyApproved'))).toBeOnTheScreen();
+            expect(screen.getByText('approved via')).toBeOnTheScreen();
```

Comment:
I don't like this change but it's necessary because `iou.automaticallyApproved` has an embedded link which is rendered as a separate component now when translations always work

Maybe you guys have some idea how to improve it?

Reply:
Hmm you could potentially split this into 3 substrings.

translateLocal('iou.automaticallyApproved') can be seen as `RAW_TEXT1` + `HTML_TAG` + `RAW_TEXT2`.

Then you could possibly check if we render:

 1.  A component with `RAW_TEXT1`
 2. A component with embedded link
 3. A component with `RAW_TEXT2`